internal/work/dispatcher.go

```diff
@@ -274,35 +273,47 @@ func (d *Dispatcher) Transmit(sender dbus.Sender, addr string, messageID string,
 			if err != nil {
 				return TransmitResponseErr, nil, nil, NewDBusError("Transmit", fmt.Sprintf("cannot read HTTP response body: %v", err))
 			}
-			resp.Body.Close()
-		}
-	}
 
-	ch := make(chan yggdrasil.Response)
-	d.Outbound <- struct {
-		Data yggdrasil.Data
-		Resp chan yggdrasil.Response
-	}{
-		Data: yggdrasil.Data{
-			Type:       yggdrasil.MessageTypeData,
-			MessageID:  messageID,
-			ResponseTo: responseTo,
-			Version:    1,
-			Sent:       time.Now(),
-			Directive:  addr,
-			Metadata:   metadata,
-			Content:    data,
-		},
-		Resp: ch,
-	}
+			err = resp.Body.Close()
+			if err != nil {
+				return TransmitResponseErr, nil, nil, NewDBusError("Transmit", fmt.Sprintf("cannot close HTTP response body: %v", err))
+			}
+			responseCode = resp.StatusCode
+			responseMetadata = make(map[string]string)
+			for header := range resp.Header {
+				responseMetadata[header] = resp.Header.Get(header)
+			}
+			responseData = data
+		} else {
+			log.Debugf("URL: '%v' has no scheme", addr)
```

Comment:
I feel like this should be an error that we pass back to the worker. If they are
configured as a RemoteContent worker, but are not sending with a value URL, we
shouldn't just log the error. We should respond with the error.